src/minimal_unified_api.py

```
@@ -0,0 +1,177 @@
+@app.post("/analyze/summarize")
+async def summarize_text(text: str):
+    """Summarize text using T5 model."""
+    try:
+        # Lazy load summarization model
+        global summarization_model
+        if summarization_model is None:
+            logger.info("Loading summarization model...")
+            try:
+                import sentencepiece  # Required for T5 tokenizer
+            except ImportError:
+                logger.error("sentencepiece not installed. Please install it: pip install sentencepiece")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)